configs/esm_software/esm_runscripts/defaults.yaml

```diff
@@ -1,11 +1,17 @@
-
 per_model_defaults:
-        file_movements:
-                default:
-                        all_directions: copy
-                bin:
-                        init_to_exp:    copy
-                        exp_to_run:     copy
-                        run_to_work:    copy
-                        work_to_run:    copy
-
+    file_movements:
+        default:
+            all_directions: copy
+        bin:
+            init_to_exp:    copy
+            exp_to_run:     copy
+            run_to_work:    copy
+            work_to_run:    copy
+default_workflow_phases:
```

Comment:
I would call it`workflow`: you're already in the `defaults.yaml` file and the workflow has the `phases` property below.

Reply:
I agree. Because it will be in the config dict section `config["general"]["defaults.yaml"]["workflow"]...`. Otherwise it would conflict with the `workflow` section in `config["general"]`.